api/serializers/non_teaching_help.py

```diff
@@ -1,9 +1,29 @@
+from django.db.models import QuerySet
 from rest_framework import serializers
 
-from api.models import NonTeachingHelp
+from api.models import NonTeachingHelp, Teacher
+from api.models.choices.non_teaching_help import NonTeachingHelpType
 
 
 class NonTeachingHelpSerializer(serializers.ModelSerializer[NonTeachingHelp]):
     class Meta:
         model = NonTeachingHelp
         fields = "__all__"
+
+
+class NonTeachingHelpPublicSerializerField(serializers.Field):  # type: ignore
```

Comment:
Not sure "Public" is needed here. Just listing types of help should be the same in "public" and "private" views. Either both non-teaching help and peer help fields should be called "public" or none of them IMO

Reply:
Removed Public here :)